performant-software/textlab
# Show tei attributes edits in stages on the transcription show

## What does this issue do?
1. Shows different transcriptions for different stages of a given document.
## How to test?
1. Login and go to https://mel-textlab.herokuapp.com/transcriptions/16903
2. Make sure the you are able to see transcriptions on the document that are different from the transcriptions you see when you click on the other menu item under the Diplomatic tab (scroll down to see the transcriptions for the second item). You should see more changes for items that appear later on the menu.
